Fly_Swatter/fire_mode.py

```diff
@@ -15,44 +15,44 @@ def check_valdity(solution):
   else:
     return True
 
```

Comment:
completely overhauled the generation of targets and vectors from the firing mode to maintain easier control and indepdence, shifted arguments away from data to a contact center approached; probably need more commenting here;